log4j-parent/pom.xml

```diff
@@ -995,6 +996,55 @@
         </executions>
       </plugin>
 
+      <plugin>
+        <groupId>org.codehaus.gmavenplus</groupId>
+        <artifactId>gmavenplus-plugin</artifactId>
+        <version>${gmavenplus-plugin.version}</version>
+        <configuration>
```

Comment:
Could you move `configuration` inside the `execution`, please?

Reply:
Fixed in https://github.com/apache/logging-log4j2/pull/2930/commits/c3db320ab279ece57e97a36a8b7cae52cdf2f338